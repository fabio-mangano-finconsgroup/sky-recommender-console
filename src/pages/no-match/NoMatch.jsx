import React from 'react';

const NoMatch = () => {
  return <>Page not found.</>;
};

export default NoMatch;
