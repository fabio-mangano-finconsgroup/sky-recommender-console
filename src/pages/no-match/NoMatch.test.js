import { render, screen } from '@testing-library/react';
import NoMatch from './NoMatch';

describe('NoMatch page', () => {
  it('should render', async () => {
    render(<NoMatch />);

    await screen.findByText('Page not found.');

    expect(document.body.childNodes[0].children).toMatchSnapshot();
  });
});
