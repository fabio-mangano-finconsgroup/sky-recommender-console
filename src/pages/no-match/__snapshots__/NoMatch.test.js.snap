// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch page should render 1`] = `HTMLCollection []`;
